README.adoc

```diff
@@ -75,12 +75,46 @@ brew tap advancedtelematic/otaconnect
 brew install aktualizr
 ----
 
+You can build and install the latest development version of aktualizr on MacOS (current head of the development branch):
+----
+brew tap advancedtelematic/otaconnect
+brew install --HEAD aktualizr-dev
+----
+
+Please, be aware that the aktualizr-dev formula is `keg only`. To make it "visible" at system level, please,
+link the development executables and libraries to /usr/local and make sure that the release version is "unlinked"
+if it's installed:
+
+----
+brew unlink aktualizr
+brew link --force aktualizr-dev
+----
+
+You can switch back to the release version by doing reverse operations:
+
+----
+brew unlink aktualizr-dev
+brew link aktualizr
+----
+
 You can also build it yourself, with basic dependencies from homebrew. You can install the necessary dependencies as follows:
 
 ----
 brew install asn1c boost cmake libarchive libsodium pkgconfig python3
 ----
 
+and run cmake followed by make in a build directory:
+
+----
+cmake .. -DWARNING_AS_ERROR=off \
+         -DBoost_USE_MULTITHREADED=ON \
+         -DOPENSSL_SSL_LIBRARY=#{Formula["openssl@1.1"].opt_prefix}/lib/libssl.dylib \
```

Comment:
Do the `#{Formula[...]...}` thing works if run outside of Homebrew?

Reply:
It shouldn't, I actually cannot compile it outside of homebrew at all due to treating warnings as errors and for some reason nothing helps me to turn it off.